src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/AmiNameResolverIT.kt

```diff
@@ -0,0 +1,96 @@
+package com.atlassian.performance.tools.awsinfrastructure
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.regions.Regions.*
+import com.atlassian.performance.tools.aws.api.Aws
+import org.junit.Test
+
+class AmiNameResolverIT {
```

Comment:
Added a test for every US, EU and AP region. Didn't add for other ones.

On my local `EU_SOUTH_1` (Milan) and `AP_EAST_1` (Hong Kong) show some authentication issue, so we might not want to check on those. I will probably remove those 2 after confirming that they fail similarly on CI.

The only region that really is failing on my local from the ones that I added tests for is `US_EAST_1`, which is very interesting.